### ISSUE TITLE
Reference dialog doesn't allow selecting full framework assemblies when Cross-targeting

### ISSUE THREAD

**Issue by BillHiebert** [dotnet/roslyn-project-system#1609]
1. Create an asp.net core app on Full Framework
2. R-click, Add Reference. Notice the choices available:
![image](https://cloud.githubusercontent.com/assets/1553785/23281355/ce6000b8-f9d1-11e6-861d-b606ab503f67.png)

3. Now create an asp.net core app. Edit project file and add a 2nd framework like net452. 
4. R-click, Add Reference. Notice the options to add assemblies (and COM) are missing:
![image](https://cloud.githubusercontent.com/assets/1553785/23281437/25af991e-f9d2-11e6-8241-6568746dbe56.png)

You need to edit the project file and add framework references manually.

**Comment by davkean** [comment]
Yep, see: https://github.com/dotnet/roslyn-project-system/issues/400#issuecomment-265622372 for how xproj did this.
